.eslintignore

```diff
@@ -0,0 +1,2 @@
+# Ignore external libraries
+js/lib/*
```

Comment:
If I use npm, how do I include the d3 library script in the viz.js file? All solutions so far use `require` or `import` (would need to transpile).